spec/system/admin/reports/enterprise_summary_fees/enterprise_summary_fee_with_tax_report_by_producer_spec.rb

```diff
@@ -133,54 +162,71 @@
     #     - outgoing exchange (10) 1.5% = 0.15, 2.5% = 0.25
     #     - line items        (50) 1.5% = 0.75, 2.5% = 1.25
 
-    context "an order with line items from a single supplier (1)" do
-      before do
-        order.line_items.create({ variant: variant, quantity: 1, price: 100 })
-        order.update!({
-                        order_cycle_id: order_cycle.id,
-                        ship_address_id: ship_address.id
-                      })
-        # This will load the enterprise fees from the order cycle.
-        # This is needed because the order instance was created
-        # independently of the order_cycle.
-        order.recreate_all_fees!
-        while !order.completed?
-          break unless order.next!
-        end
+    before do
+      # adds a line items to the order on oc1
+      order.line_items.create({ variant: variant, quantity: 1, price: 100 })
+      order.update!({
+                      order_cycle_id: order_cycle.id,
+                      ship_address_id: ship_address.id
+                    })
+      # This will load the enterprise fees from the order cycle.
+      # This is needed because the order instance was created
+      # independently of the order_cycle.
+      order.recreate_all_fees!
+      while !order.completed?
+        break unless order.next!
+      end
```

Comment:
This should be moved to a helper function it's used multiple time in the spec file